test/e2e/tests/confirmations/transactions/speed-up-and-cancel-confirmations.spec.ts

```diff
@@ -28,10 +27,9 @@ describe('Speed Up and Cancel Transaction Tests', function () {
             .withPermissionControllerConnectedToTestDapp()
             .build(),
           localNodeOptions: defaultGanacheOptionsForType2Transactions,
-          smartContract: SMART_CONTRACTS.PIGGYBANK,
```

Comment:
this is not needed in this spec, as we are not interacting with any smart contract